openroad_interface/test_files/mm_test.gml

```diff
@@ -0,0 +1,797 @@
+graph [
```

Comment:
NOw that this is being integrated into the codesign repo, we should have these tests point to `codesign/src/architectures/<filename>.gml` instead of having a copy in here